prefab/dependency-integration/README.md

```diff
@@ -0,0 +1,87 @@
+Integrating Prefab Dependency
+==============================
+
+This sample shows how to import [C/C++ dependencies] from Google Maven to your Android project.
+Those dependencies are C/C++ libraries packed with [Prefab] format into an AAR.
```

Comment:
Those dependencies are C/C++ headers and packages packaged inside an AAR using the Prefab package format.